### DISCUSSION TITLE
Can not change SolaX Backup Discharge Min SOC

### DISCUSSION THREAD

**Discussion by jan-bee** [wills106/homeassistant-solax-modbus#119] [General]
Hi,
I tried to set new "SolaX Backup Discharge Min SOC" by slider, but with no success.
entity: number.solax_backup_discharge_min_soc

Change of other values (like number.solax_export_control_user_limit or number.solax_selfuse_discharge_min_soc) working fine. 
Inverter is running in Backup mode, because I want to prevent battery from discharging in some night times. In Self Use mode inverter go to "Idle" and it is not possible to wake it. 

version 0.6.2 (same in 0.6.1)
HA 2022.9.5 (same in 2022.8.5)
SN: H34T (Solax G4 10K)

Do you have any hint?

Thank you.
Jan

**Comment by bo4arov** [comment]
Greetings, Indeed, this setting is not being applied. I change 30 -> 20 and it restores to 30 in a few seconds.  
It worked from Solax Cloud interface though, so it has something to do with how integration sets the value.

**Reply by rosenrot00** [reply]
I just changed it to 39% and it worked. Could you open an issue and upload a debug log?
